src/vivarium/framework/engine.py

```diff
@@ -208,7 +206,7 @@ def name(self) -> str:
         return self._name
 
     def get_results(self) -> Dict[str, Any]:
-        return self._values.get_value("metrics")(self.get_population().index)
+        return self._results.metrics
```

Comment:
Remind me - the plan is for the next PR to do the formatting here, right?

Reply:
Yes